src/layout/Header/index.tsx

```diff
@@ -13,60 +14,95 @@ const Header = () => {
-                  <DropDownMenu menuContent={['Sprint →', 'Audio-call →']} />
-                </div>
-              )}
+    <div>
+      <header className="flex h-[88px] items-center justify-center bg-tk_greyLight shadow-xl">
+        <div className="flex w-full items-center justify-between px-[8px] md:px-[32px] lg:w-[1180px] lg:px-0">
+          <nav className="flex items-center gap-[30px]">
+            <h1 className="font-Dela_Gothic_One text-[21px] font-normal tracking-[-0.04em]">
+              TaoTalk
+            </h1>
+            <div className="hidden md:flex md:gap-[30px]">
+              <span className="text-tk_greyMedium">|</span>
+              <ul className="flex list-none items-center pl-0">
+                <li>
+                  <a
+                    title="Home"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-black no-underline"
+                  >
+                    Home
+                  </a>
+                </li>
+                <li>
+                  <a
+                    title="Textbook"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Textbook
+                  </a>
+                </li>
+                <li>
+                  <a
+                    title="Statistics"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Statistics
+                  </a>
+                </li>
+                <li
+                  className="relative"
+                  onMouseEnter={handleMouseEnter}
+                  onMouseLeave={handleMouseLeave}
+                >
+                  <a
+                    title="Games"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Games
+                    <img src="/down_arrow.svg" alt="down array"></img>
+                  </a>
+                  {isHovered && (
+                    <div className=" absolute top-[46px] w-[215px] rounded-[14px] bg-white shadow-[0_343px_343px_0_rgba(23,64,51,0.08)] ">
+                      <DropDownMenu menuContent={['Sprint →', 'Audio-call →']} />
+                    </div>
+                  )}
+                </li>
+              </ul>
+            </div>
+          </nav>
+          <div className="flex gap-8 lg:hidden">
+            <LogInButton userName={'Alex'} isSimplified={true} />
+            <div className="flex md:hidden">
+              <MenuButton isMenuShown={isMenuShown} onClick={handleMouseClickOnMenuButton} />
```

Comment:
![code](https://github.com/taodemy/taotalk/assets/76163407/2c17f47a-b040-48d4-98f8-873dd53c5175)
there is no need to make a component, just use the above code directly

Reply:
https://github.com/taodemy/taotalk/blob/d8f609ef26d9a3dc290c100b53baff68a5dfc43a/src/layout/Header/index.tsx#L66-L74

The component has been replaced by the above codes to make files simpler.